setup.py

```diff
@@ -19,7 +19,9 @@
     install_requires=[
         "PyYAML>=3.13",
         "pandas>=0.19.2",
-        "requests>=2.21.0"
+        "requests>=2.21.0",
+        "fs<2.5.0",
+        "fs.archive<0.7.0"
```

Comment:
why is set `fs<2.5.0` and `fs.archive<0.7.0` instead of `fs>=2.5.0` and `fs.archive>=0.7.0`?

Reply:
because I believe that major versions break some contracts

Reply:
I'm limiting for just bug fixes